src/locomoset/metrics/task_agnostic.py

```diff
@@ -0,0 +1,59 @@
+"""
+Functions for the task-agnostic metrics. In particular the number of parameters of a
+model as a metric.
+
+This also includes functions for reweighting a set of metric values by the inclusion of
+a task agnostic metric as a proxy for 'capactity to learn', as introduced in the
+Pairwise Annotation Representation Comparison (PARC) metric for transferability paper:
+
+Bolya, Daniel, Rohit Mittapalli, and Judy Hoffman. "Scalable diverse model selection
+for accessible transfer learning." Advances in Neural Information Processing Systems
+34 (2021): 19301-19312.
+"""
+
+import numpy as np
+from transformers import AutoModelForImageClassification
+
+
+def num_params_metric(model: AutoModelForImageClassification) -> int:
+    """Return the number of parameters of a model as a metric.
+
+    Args:
+        model: pre-loaded hugging face model from which to return the number of
+        parameters.
+
+    Returns:
+        number of paramters of said model.
+    """
+    return model.num_parameters()
+
+
+def rescale_by_params(models_and_metrics: dict) -> dict:
```

Comment:
Having a test for this would be good

Reply:
A test for `rescale_by_params` in particular?

Reply:
If so I hadn't written that yet as I hadn't considered it finished.

Reply:
yeah for `rescale_by_params`, no probs 👍